plugins/namelayer-paper/src/main/java/vg/civcraft/mc/namelayer/command/commands/DeleteGroup.java

```diff
@@ -29,36 +29,40 @@ public void execute(Player sender, String groupName) {
 		UUID uuid = NameAPI.getUUID(p.getName());
 		String x = groupName;
 		String confirm = "CONFIRM_DELETION";
-		if(x.equals(confirm))
+		if(x.contains(confirm))
```

Comment:
Might wanna be more considerate towards users by accepting both uppercase and lowercase, i.e. comparing with .toLowerCase() or equivalent.

Reply:
Fixed